File: moziretusyori/paiza3.py
N = int(input())
Q = int(input())
password = ['A'] * N

for i in range(Q):
    a, b = input().split()
    a = int(a)
    password[a - 1] = b

C = input()

for i in range(N):
    if password[i] == 'A':
        password[i] = C

print(*password, sep='')

File: morinocontest9/paiza3.swift
let input = readLine()!.split(separator: " ").map { Int($0)! }
let n = input[0]
let k = input[1]
let a = readLine()!.split(separator: " ").map { Int($0)! }

let rangeSumArray : [Int] = []

for i in 0..<n - k + 1 {
    var sum = 0
    for j in i..<i + k {
        sum += a[j]
    }
    rangeSumArray.append(sum)
}

print(rangeSumArray.max()!)